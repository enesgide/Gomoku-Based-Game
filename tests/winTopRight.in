place S1
place S2
place R2
place R3
place Q3
place Q4
place P4
place P5
place O5