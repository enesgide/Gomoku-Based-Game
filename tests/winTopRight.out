Black wins!
S1S2R2R3Q3Q4P4P5O5
Thank you for playing!
